cobra/test/test_manipulation.py

```diff
@@ -216,12 +216,47 @@ def test_validate_mass_balance(self, model):
             r1.check_mass_balance()
 
     def test_prune_unused(self, model):
+
+        # test that the output contains metabolite objects
```

Comment:
Please make a separate test case (function) for each section that you start with a comment.